### DISCUSSION TITLE
如何同时显示双轨字幕

### DISCUSSION THREAD

**Discussion by newyorker123** [hooke007/mpv_PlayKit#418] [Q&A 问答]
请问一下，比如日剧有日语字幕和中文字幕两轨，如何能同时显示呢？

如果目前无法实现就算了，主要怕有这个功能但我没有发现。

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/7e66c6b9873fa1f567da1f886d435a9efe7b44ec/portable_config/input_easy.conf#L119-L120
样式问题看faq
